### DISCUSSION TITLE
[Desktop] StrokeDashArray not working

### DISCUSSION THREAD

**Discussion by kucint** [unoplatform/uno.extensions#2315] [Q&A] [answered]
Following code is expected to produce dashed ellipse:
```cs
    public MainPage()
    {
        this
            .Background(ThemeResource.Get<Brush>("ApplicationPageBackgroundThemeBrush"))
            .Content(new StackPanel()
            .VerticalAlignment(VerticalAlignment.Center)
            .HorizontalAlignment(HorizontalAlignment.Center)
            .Children(
                new TextBlock()
                .Text("Hello Uno Platform!"),
                new Ellipse()
                .Width(100)
                .Height(100)
                .Stroke(Colors.DeepPink)
                .StrokeThickness(1)
                .StrokeDashArray(new double[] { 2, 2 })
            ));
    }
```

under "desktop" StrokeDashArray is ignored:
![image](https://github.com/unoplatform/uno.extensions/assets/54866088/3a5f97fc-5442-4687-9bdb-6039d34e4d59)

while under net8.0-windows10.0.19041 all is ok
![image](https://github.com/unoplatform/uno.extensions/assets/54866088/a6984f6b-93c9-42b7-aaf1-4e358b15dc57)

tested under windows only.
```json
{
  "msbuild-sdks": {
    "Uno.Sdk": "5.2.139"
  }
}

```

project attached: [UnoStrokeDashApp.zip](https://github.com/user-attachments/files/15531598/UnoStrokeDashApp.zip)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This is likely fixed in the latest prerelease builds of uno.sdk (https://github.com/unoplatform/uno/pull/16702).
